### DISCUSSION TITLE
要么退款，要么解决问题！！不然我会一直发帖，到处投诉举报！！！

### DISCUSSION THREAD

**Discussion by JMaster1937** [getlantern/lantern-forum-cn#805] [General]
买了一年了根本收不到验证码，无法正常使用，要么退款，要么解决问题！！不然我会一直发帖，到处投诉举报！！！
![6dccab6007ea6d280745a704d33cd0a](https://github.com/user-attachments/assets/0f427a04-ca77-41f2-b712-46a9fd8477ac)

**Comment by MaxMKMax** [comment]
@JMaster1937 您好. 因为国内的围墙的关系所以影响到了验证码的发送. 
请您也提供我您的账号邮箱. 我帮您提交问题. 请您息怒, 别生气. 谢谢您的耐心等待.

**Reply by JMaster1937** [reply]
支付宝支付单号为：2024080622001303891444899483 账号：1150828904@qq.com 如国内限制原因可将我的会员转到 邮箱为：ma4241776@gmail.com 的账号上 请尽快解决！

**Reply by wander0324** [reply]
> @JMaster1937 您好. 因为国内的围墙的关系所以影响到了验证码的发送. 请您也提供我您的账号邮箱. 我帮您提交问题. 请您息怒, 别生气. 谢谢您的耐心等待.

你好，我8月16日购买的1年专业版，账号[85423166@qq.com](mailto:85423166@qq.com)，订单号2024081622001316131454810921，目前的问题是登不上专业版，依然显示免费版，麻烦帮忙查看并解决呢，谢谢！
